src/components/PerformanceRanking/PerformanceRanking.tsx

```
@@ -0,0 +1,275 @@
+ */
+export interface PerformanceRankingData {
+  groupedBy: GroupedBy;
+  highlighted: (PerformanceRankingItem | null)[];
+  needsAttention: (PerformanceRankingItem | null)[];
+}
+
+/**
+ * Props for the PerformanceRanking component
+ */
+export interface PerformanceRankingProps
+  extends HTMLAttributes<HTMLDivElement> {
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)